src/app/state/rewardUtils.tsx

```diff
@@ -0,0 +1,399 @@
+import { TokenInfo } from "alphadex-sdk-js";
+import { getRdtOrThrow } from "../subscriptions";
+import { StateKeyValueStoreDataRequestKeyItem } from "@radixdlt/radix-dapp-toolkit";
+
+export class ClaimComponent {
```

Comment:
Is there any practical reasons for using classes instead of interfaces here?

Reply:
I am not sure, but it is often easier to have a class with initial values that you can then change with imported values, but I will need to check the code. 
I would suggest we put in a separate issue for addressing some of these comments. I do not want to make any further changes to the code unless it is a bug at this stage.
I will create a new issue called "Rewards page enhancements" where I will add these comments and we can deal with them there after the launch.